modules/setting/repository.go

```diff
@@ -282,6 +282,7 @@ func loadRepositoryFrom(rootCfg ConfigProvider) {
 	} else {
 		RepoRootPath = filepath.Clean(RepoRootPath)
 	}
+	Repository.AccessControlAllowOrigin = strings.ToLower(sec.Key("ACCESS_CONTROL_ALLOW_ORIGIN").MustString(""))
```

Comment:
I have tested, this line is not needed. Because there is a `sec.MapTo(&Repository)` below, these fields will be auto-filled.